packages/generator/src/subtask2/my-func.test.ts

```diff
@@ -0,0 +1,33 @@
+import {createNewColumn} from './my-func';
+import ts from 'typescript';
+test('my own test', () => {
+    createNewColumn(` 
+
+    import{ aaa, asd,TableColumn} from 'dsada';
+    import{ ss} from 'dsada';
+    
+    
+
+    <Table>
```

Comment:
Please simplify the example/input. It would be better to just have only two columns.